app/assets/javascripts/editor/MapOverlays/GeoCATMarker.js

```diff
@@ -349,7 +349,10 @@
       /* Set active property of the occurrence */
       GeoCATMarker.prototype.setActive = function(active) {
         this.data.geocat_active = active;
-        this.canvas_.style.opacity = (active)?1:0.3;
+        // if (! !!sessionStorage.getItem('toggleing_global'))
```

Comment:
I know that there's already enough of that throughout the app :p so let's try not to add more =D